doc/adr/0014-branch-names-and-commit-messages.md

```diff
@@ -0,0 +1,41 @@
+# 14. Branch names and commit messages
+
+Date: 2022-11-28
+
+## Status
+
+Accepted
+
+## Context
+
+The tasks of this project need to be distributed over the actively working employees of DigitalService. But when many people work concurrently on different tasks of a project in git, there might be a lot of commits and branches, so one could lose overview.
```

Comment:
Since this working agreement is initiated in our team, I guess it's not `employees of DigitalService` but rather team members?